index.html

```diff
@@ -8,47 +8,324 @@
+            <li><a href="#news">News</a></li>
+            <li><a href="#contact">Contact</a></li>
+            <li>
+              <a href="#about">
+                <i class="fas fa-envelope"></i>
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="humberger">
+          <i class="fa fa-bars"></i>
+        </div>
       </nav>
-    </header>
-    <section class="headline-page">
-      <div class="introduction">
-        <h1 class="intro">Hey There.<br />I'm Mucyo</h1>
-        <h2 class="headline-page-h2">I'am a Software Developer</h2>
-
-        <p class="text-1">
-          I can help you build a product, feature or website <br />Look through
-          some of my work and experience!<br />
-          if you like what you see and have a project <br />
-          you need coded, don't hestiate to contacta me.
-        </p>
+      <main>
+        <div class="header">
+          <h1>Hey There.<br />I'm Mucyo</h1>
+          <h2>I'am a Software Developer</h2>
+
+          <p>
+            I can help you build a product, feature or website <br />Look
+            through some of my work and experience!<br />
+            if you like what you see and have a project <br />
+            you need coded, don't hestiate to contacta me.
+          </p>
+        </div>
+        <div id="social-icons">
+          <ul>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-medium" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-github" aria-hidden="true"></i></a>
+            </li>
+            <li class="fa-size">
+              <a href="#"><i class="fa fa-angellist" aria-hidden="true"></i></a>
+            </li>
+          </ul>
+        </div>
+      </main>
+      <div id="content_title">
+        <div class="my_recent">
+          <h1>My Recent Works</h1>
+        </div>
+        <div class="line1"></div>
+      </div>
+
+      <div id="sidebar">
+        <div class="txt_image">
+          <img src="images/Img.png" alt="image" class="middle-image" />
+        </div>
+        <div class="post1">
+          <h1>Multi-Post Stories</h1>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul class="links1">
+            <li><a href="#"></a>css</li>
+            <li><a href="#"></a>html</li>
+            <li class="boot"><a href="#"></a>bootstrap</li>
+            <li><a href="#"></a>Ruby</li>
+          </ul>
```

Comment:
[ OPTIONAL ] Please follow the font color and font family defined in the design.